### ISSUE TITLE
Min Browser won't install

### ISSUE THREAD

**Issue by Grejuva** [minbrowser/min#853]
I'm on a Windows OS computer, but the "Download" button on the website gives me a file with a .deb extension. The other zips and .exe files I found around wont install as well, giving me several JavaScript errors during the installation, and opening an un-interactive browser window in the end. Help?

**Comment by PalmerAL** [comment]
What version of Windows are you using? Is it 32- or 64-bit? (on Windows 10, you can find out by going to settings > system > about > system type)

Also, could you go here: https://www.whatismybrowser.com/detect/what-is-my-user-agent, and post the text that appears in the blue box?

Finally, could you take a screenshot of the errors you get when you install the exe file, as well as what the browser looks like after it's installed?

Thanks!

**Comment by Grejuva** [comment]
Sorry for the late response. It's 64-bit Windows 10. And here's the error message I get. This error pops up once as the installer is halfway through the installation and once again before an uninteractive browser window opens. Am I missing something?
My user agent:
![error1](https://user-images.githubusercontent.com/46448819/69005575-28c2e780-0935-11ea-903f-5fe0926340a6.PNG)

**Comment by PalmerAL** [comment]
The user-agent looks OK; I'm wondering if the fact that Puffin is rendering the page remotely is causing this? If they're not setting `navigator.platform` correctly, we could be detecting whatever platform they're rendering the page on rather than the platform of your local computer.

For the second error, can you open this file:

```
C:\Users\username\AppData\Roaming\Min\settings.json
```
and post the contents here? And once you do that, try deleting that file and then reinstalling.

**Comment by Grejuva** [comment]
Oh yes, downloading with Firefox gave me the .exe version.
As for that file, "    {"darkMode":true,"filtering":{"trackers":true}}}}   "  was the content when I opened it with notepad. Deleted it, and I got no error during reinstallation. Instead, opening the browser gave the that error and the same unresponsive browser window. With the browser installed, I decided to delete everything in that folder. I don't know how or why, but it worked just like that, my problem is solved, thanks.
